private/bufpkg/bufcheck/bufbreaking/bufbreakingconfig/bufbreakingconfig.go

```diff
@@ -98,6 +99,21 @@ func ProtoForConfig(config *Config) *breakingv1.Config {
 	}
 }
 
+// ValidateProtoConfig validates that the proto config is valid.
+func ValidateProtoConfig(protoConfig *breakingv1.Config) error {
+	// An empty config is valid.
+	if protoConfig == nil {
+		return nil
+	}
+	version := protoConfig.GetVersion()
+	switch version {
+	case v1Version, v1Beta1Version:
+		return nil
+	default:
+		return fmt.Errorf("invalid breaking config version %q provided", version)
+	}
+}
+
```

Comment:
I don't know if we want to validate on the client-side. This is how we ended up in that issue about requiring fields in older versions of the CLI, but changing that requirement on the server-side later (the `v0.43.0` -> `v0.44.0` change). We need to think through this.